.github/workflows/qat.yml

```diff
@@ -16,74 +16,64 @@ on:
 
     secrets:
       BITBUCKET_READ_ONLY_SSH_KEY:
-        required: true
+        required: false
       GITHUB_READ_ONLY_SSH_KEY:
+        required: false
+      CHECKLY_API_KEY:
+        required: true
+      CHECKLY_ACCOUNT_ID:
         required: true
```

Comment:
If these are omitted, does the QAT job fail? abort/not run? 

I'm thinking of themes which don't have QAT yet. An old step being deleted below `name: Check QAT Script and Feature Branch` previously did a check for the presence of qat scripts, I understood this to mean that this qat.yml was running on all themes already? Is that not the case? ..and will only be run if the theme has `uses: penske-media-corp/github-workflows-wordpress/.github/workflows/qat.yml`?

Reply:
E.g. from WWD which does not have QAT yet https://github.com/penske-media-corp/pmc-wwd-2021/pull/79

![Screen-Shot-2023-10-05-at-4-19-11-PM](https://github.com/penske-media-corp/github-workflows-wordpress/assets/491189/a3de8cbd-1d5b-47fc-9d7c-8cdcdc622d65)

Reply:
Good catch. The Playwright setup had a step that bailed early if the theme hadn't configured QAT. We need to restore that. 

All themes (and `pmc-plugins`) reference this QAT workflow, but only SJ configures it, so it bails early everywhere else.

Reply:
Can qat NOT be run on those themes? Ie can we disable that behavior elsewhere?

Reply:
Not without updating the workflow file in every theme. It was intentionally included [as part of the template](https://github.com/penske-media-corp/github-workflows-wordpress/blob/751d6549c12ba335bf1c9cbbb2759af3749dfda8/templates/pmc.yml) so that we didn't need to update every repo that uses these workflows.

Reply:
I see.. 

https://github.com/search?q=org%3Apenske-media-corp%20uses%3A%20penske-media-corp%2Fgithub-workflows-wordpress%2F.github%2Fworkflows%2Fqat.yml%40main&type=code

Reply:
Thanks both! I added those checks back in.